snap/bin/start-config-seed.sh

```diff
@@ -32,12 +30,9 @@ done
 #
 # TODO: this success check could be improved...
 if [ $CONSUL_RUNNING != "[]" ] ; then
-    cd $SNAP/jar/config-seed/
-
-    $JAVA -jar -Djava.security.egd=file:/dev/urandom -Xmx100M \
-        $SNAP/jar/config-seed/core-config-seed.jar
-
+    cd $SNAP/config/core-config-seed-go
 
+    $SNAP/bin/core-config-seed-go -consul &
```

Comment:
@tonyespy This line might be incorrect. The flag should be "--consul". Two dashes. Alternately, you could use "-c". Please confirm you've seen this work.